website/data/docs-nav-data.json

```diff
@@ -712,6 +712,10 @@
             "title": "Image",
             "path": "datasources/hcp/hcp-packer-image"
           },
+          {
```

Comment:
Here, you put the datasource under the HCP datasources. You need to move this one up and update the path to accordingly to the changes I proposed above `datasources/http`